tests/cmd-args.py

```diff
@@ -5,7 +5,7 @@
 
 failed = False
 
-for f in filter(lambda x: x.startswith('llvm'), listdir()):
+for f in filter(lambda x: x.startswith('llvm') and not x.endswith('.cpp'), listdir()):
```

Comment:
Maybe we should add a check for the executable bit instead?

Reply:
The x bit could be set even for source files without having any impact... but I guess then that it is a problem of the user that would mess with the bits. Yes, le't go with checking the x bit.